### ISSUE TITLE
Return PIN along with chat link

### ISSUE THREAD

**Issue by muke1908** [muke1908/chat-e2ee#152]
This allows user to share the short PIN instead of long chat link. Other user can directly join the chat by entering PIN.

Add following to response object:
```
pin, pinCreatedAt
```

https://github.com/muke1908/chat-e2ee/blob/169a1d984fd8e92744ae1de46eca4a2e6ca65fb4/backend/api/chatLink/utils/link.js#L5

Blocked on: https://github.com/muke1908/chat-e2ee/issues/150

For now dummy function can be used to get 4 digit PIN

**Comment by abhishek-aa** [comment]
Correct me if I am wrong but can't two chat links result in same 4 digit PIN?

**Comment by muke1908** [comment]
@abhishek-aa will be using this[ function ](https://github.com/muke1908/chat-e2ee/issues/150) to make sure that the PIN is unique for every chat link. We get around 1679616  combination (alpha numeric case in-sensitive) which is pretty decent.

**Comment by Narasimha1997** [comment]
Are you planning to add PIN expiry?

**Comment by muke1908** [comment]
Hi, @Narasimha1997  yes, PINs will be considered as expired after a certain period of time. Will create a separate issue for that.

**Comment by Narasimha1997** [comment]
Maybe we can close this.
